### ISSUE TITLE
Put AOI transitions into their own file as table

### ISSUE THREAD

**Issue by ashkjones** [TheD2Lab/gazepoint-data-analysis#23]
The current method of appending transitions to the DGMs is poorly designed. It makes it hard to parse the data because the headers are different in every file. I have to use pattern matching to find the columns I need, which adds an additional level of complexity. 

The information on which pair the data belongs to relies on looking at two cells. Its not even the same two cell for a pair across multiple participant files. This requires a whole algorithm to determine which pair the data belongs to. 

Putting aside that I need to parse this in a program, the current design is hard for a human to read. Please make a new file for AOI transitions that has 4 headers as follows `AOI Pair`, `Transition Count`, `Proportion excluding self-transition`, and `Proportion including self-transition`.

**Comment by Immersor-git** [comment]
Changed output format to desired style - outputs can be found in the file named participantName_AOI_Transitions. Fix is currently on the aoi-pairs-revision branch.
